source/getting-started/faq.rst

```diff
@@ -248,6 +249,22 @@ ________________________________________
 
 The `Chameleon Appliance Catalog <https://www.chameleoncloud.org/appliances/>`_ is a repository that allows users to discover, publish, and share appliances. The appliance catalog contains useful images of both bare metal and virtual machine appliances supported by the Chameleon team as well appliances contributed by users.
 
+.. _how-build-appliance:
+
+How to build or customize a Chameleon appliance?
+_________________________________________________________________
+
+There are two options to build or customize a Chameleon appliance -- the ``cc-snapshot`` utility and the OpenStack ``diskimage-builder``.
+
+:ref:`cc-snapshot-utility`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The ``cc-snapshot`` tool is pre-installed in all Chameleon supported appliances and it provides a quick and easy way to customize a Chameleon appliance. To start, spin up an instance with the Chameleon appliance you would like to customize.  Then install the libraries and tools you would like to add into your new appliance, or uninstall things you want to exclude from your new appliance. Finally, take a snapshot!
+
+The OpenStack ``diskimage-builder``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can use ``diskimage-builder`` to build your appliance from scratch. The OpenStack ``diskimage-builder`` provides a more manageable way of building appliances. For more information about OpenStack ``diskimage-builder``, please reference the `OpenStack documentation <https://docs.openstack.org/diskimage-builder/latest/>`_.
```

Comment:
You should also add pointers to the CC-CentOS7, CC-Ubuntu16.04, and CC-Ubuntu14.04 repositories.